site/en/tutorials/structured_data/imbalanced_data.ipynb

```diff
@@ -345,11 +345,11 @@
         "pos_df = pd.DataFrame(train_features[ bool_train_labels], columns=train_df.columns)\n",
         "neg_df = pd.DataFrame(train_features[~bool_train_labels], columns=train_df.columns)\n",
         "\n",
-        "sns.jointplot(pos_df['V5'], pos_df['V6'],\n",
+        "sns.jointplot(x = pos_df['V5'], y = pos_df['V6'],\n",
```

Comment:
Assuming this is to prevent the following seaborn `FutureWarning`:

```
/usr/local/lib/python3.7/dist-packages/seaborn/_decorators.py:43: FutureWarning: Pass the following variables as keyword args: x, y. From version 0.12, the only valid positional argument will be `data`, and passing other arguments without an explicit keyword will result in an error or misinterpretation.
```

Reply:
Yap